### ISSUE TITLE
Android release build assembling fails with react-native@0.62.2

### ISSUE THREAD

**Issue by oleksandr-dziuban** [react-native-community/progress-view#31]
# Bug

Android release build fails with react-native@0.62.2

## Environment info

`react-native info` output:

```bash
System:
    OS: macOS 10.15.4
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 509.36 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.2 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 23, 25, 26, 27, 28, 29
      Build Tools: 27.0.3, 28.0.3, 29.0.3
      System Images: android-24 | Google Play Intel x86 Atom, android-25 | Google APIs Intel x86 Atom, android-26 | Google APIs Intel x86 Atom, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6308749
    Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_252 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0 
    react-native: 0.62.2 => 0.62.2 
  npmGlobalPackages:
    *react-native*: Not Found

```

Library version: 1.0.2

## Steps To Reproduce

1. Generate fresh react-native empty project
2. Install react-native-community/progress-view library
3. Run assembling release android build (build-tools v29.0.3, android 29)

Describe what you expected to happen:

1.  Release android build is assembled successfully

## Error log

```bash
2: Task failed with an exception.
 
-----------
 
* What went wrong:
 
Execution failed for task ':react-native-community_progress-view:verifyReleaseResources'.
 
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
 
   > Android resource linking failed
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/a57f657f9eb12bd95b25a6d72d815ff5/appcompat-1.0.2/res/values-v26/values-v26.xml:5:5-8:13: AAPT: error: resource android:attr/colorError not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/a57f657f9eb12bd95b25a6d72d815ff5/appcompat-1.0.2/res/values-v26/values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/a57f657f9eb12bd95b25a6d72d815ff5/appcompat-1.0.2/res/values-v26/values-v26.xml:13:5-16:13: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/a57f657f9eb12bd95b25a6d72d815ff5/appcompat-1.0.2/res/values-v28/values-v28.xml:5:5-8:13: AAPT: error: resource android:attr/dialogCornerRadius not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/a57f657f9eb12bd95b25a6d72d815ff5/appcompat-1.0.2/res/values-v28/values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:57:5-88:25: AAPT: error: resource android:attr/fontStyle not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:57:5-88:25: AAPT: error: resource android:attr/font not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:57:5-88:25: AAPT: error: resource android:attr/fontWeight not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:57:5-88:25: AAPT: error: resource android:attr/fontVariationSettings not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:57:5-88:25: AAPT: error: resource android:attr/ttcIndex not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:89:5-125:25: AAPT: error: resource android:attr/startX not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:89:5-125:25: AAPT: error: resource android:attr/startY not found.
 
         
 
     /Users/distiller/.gradle/caches/transforms-2/files-2.1/b1035ff6f51c7705ab78936f582a5533/core-1.0.1/res/values/values.xml:89:5-125:25: AAPT: error: resource android:attr/endX not found.
 
         
 
```

**Comment by oleksandr-dziuban** [comment]
@Naturalclar Could you please check this issue? Thanks a lot

**Comment by Naturalclar** [comment]
@oleksandr-dziuban sorry for the delay! I've released `1.0.3` with the fix
